babybuddy/management/commands/createuser.py

```diff
@@ -0,0 +1,192 @@
+# -*- coding: utf-8 -*-
+"""
+Management utility to create users
+
+Example usage:
+
+  manage.py createuser \
+          --username test     \
+          --email test@test.test \
+          --is-staff \
+          --is-superuser
+"""
+import sys
+import getpass
+
+from django.contrib.auth import get_user_model
+from django.contrib.auth.password_validation import validate_password
+from django.core import exceptions
+from django.core.management.base import BaseCommand, CommandError
+from django.db import DEFAULT_DB_ALIAS
+from django.utils.functional import cached_property
+from django.utils.text import capfirst
+
+
+class NotRunningInTTYException(Exception):
+    pass
+
+
+class Command(BaseCommand):
+    help = "Used to create a user"
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.UserModel = get_user_model()
+        self.username_field = self.UserModel._meta.get_field(
+            self.UserModel.USERNAME_FIELD
+        )
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--database",
+            default=DEFAULT_DB_ALIAS,
+            help='Specifies the database to use. Default is "default".',
+        )
+        parser.add_argument(
+            f"--{self.UserModel.USERNAME_FIELD}",
+            help="Specifies the login for a user.",
+        )
+        parser.add_argument(
+            "--password",
+            dest="password",
+            help="Specifies the password for the user. Optional.",
+        )
+        parser.add_argument(
+            "--is-staff",
+            dest="is_staff",
+            action="store_true",
+            default=False,
+            help="Specifies the staff status for the user. Default is False.",
+        )
+        parser.add_argument(
+            "--in-active",
+            dest="is_active",
+            action="store_false",
+            default=True,
+            help="Specifies the active status for the user. Default is True.",
+        )
+        parser.add_argument(
+            "--is-superuser",
+            dest="is_superuser",
+            action="store_true",
+            default=False,
+            help="Specifies the superuser status for the user. Default is False.",
+        )
```

Comment:
Lets actually _not_ expose super user here -- as we don't in the user add form either. Staff status is the most any user needs for this app.

Reply:
Right, I was contemplating that one as well. Will fix that!